src/main/java/frc/robot/commands/rollerarm/LogRollerArmCurrent.java

```diff
@@ -0,0 +1,49 @@
+
+package frc.robot.commands.rollerarm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.consoles.Logger;
+import frc.robot.subsystems.RollerArm;
+
+// This command logs the current being used by the roller arm
+public class LogRollerArmCurrent extends CommandBase {
+
+    private RollerArm m_rollerArm;
+
+    public LogRollerArmCurrent(RollerArm rollerArm) {
+        Logger.setup("Constructing Command: LogRollerArmCurrentInAmps...");
+
+        // Add given subsystem requirements
+        m_rollerArm = rollerArm;
+        addRequirements(m_rollerArm);
+    }
+
+    @Override
+    public void initialize() {
+        Logger.action("Initializing Command: LogRollerArmCurrentInAmps...");
+    }
+
+    @Override
+    public void execute() {
+        int current = m_rollerArm.getCurrent();
+        Logger.ending("RollerArm compressor current in amps: " + current);
```

Comment:
The log is going to be spammed with this info. Is that the intent, or should it log less frequently?